server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java

```diff
@@ -175,7 +175,7 @@ void testWhenReplacedLengthIsMoreThanReplaceable() {
             "1",
             new Diagnostic(
                 new Range(),
-                "Max length limit of 30 bytes allowed for variable name.",
+                "Max length limit of 30 characters allowed for variable name.",
```

Comment:
Maximum length of 30 characters for variable names.
Variable names cannot exceed 30 characters.

Reply:
updated